app/policies/dashboard_policy.rb

```diff
@@ -0,0 +1,7 @@
+class DashboardPolicy < ApplicationPolicy
+  VIEW_INBOX_ROLES = %w[superadmin policy_and_abuse].freeze
+
+  def can_view_inbox_link?
```

Comment:
Since this is the link and also viewing the inbox in general, I'd make this method name a little bit more generic
```suggestion
  def can_view_inbox?
```

Reply:
I was wondering whether it would make sense to combine the two policies into one by removing this one and checking `policy(:inbox_comment).show?` in the view instead. Other similar policies appear to be combined like this.

Reply:
Bilka's suggestion should be fine! But just to jump on here, we'd generally prefer to use terms like `access` over `view` -- we try to avoided using sight-centered language whenever possible, even in the code.